app/src/sharedTest/java/org/oppia/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -83,6 +85,7 @@ class ProfileEditActivityTest {
     }
   }
 
+  // Failing in other Devices too. ActionBar does not have the text "Sean". It has "Oppia"
   @Test
   fun testProfileEditActivity_configurationChange_startActivityWithAdminProfile_checkAdminInfoIsDisplayed() {
     ActivityScenario.launch<ProfileEditActivity>(ProfileEditActivity.createProfileEditActivity(context, 0)).use {
```

Comment:
This test is failing in other devices too. ActionBar does not have the text "Sean". It has "Oppia"